control-plane/agents/core/src/volume/mod.rs

```diff
@@ -15,18 +9,11 @@ pub mod specs;
 
 pub(crate) fn configure(builder: common::Service) -> common::Service {
     let registry = builder.get_shared_state::<Registry>().clone();
+    let new_service = service::Service::new(registry);
```

Comment:
Like we have done with the other services, can we just create the `Arc` here and then just pass it to `VolumeServer::new` below?